### PR TITLE
Mention the need to include icon

### PR DESCRIPTION
**Pull request by nclandrei**
Issue #8366 - This pull request informs the user regarding the need to supply an icon in part 8 of the tutorial as well as in add a manifest file page of the documentation.

### CODE REVIEW

File: docs/docs/add-a-manifest-file.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ Quoting [Google](https://developers.google.com/web/fundamentals/web-app-manifest
 npm install --save gatsby-plugin-manifest
 ```
 
-2.  Add the plugin to the `plugins` array in your `gatsby-config.js` file.
+2.  Add the plugin to the `plugins` array in your `gatsby-config.js` file (don't forget to add the icon yourself as we don't supply one).
```
Comment (yeskunall) [inline docs/docs/add-a-manifest-file.md]:
```suggestion
2.  Add the plugin to the `plugins` array in your `gatsby-config.js` file (make sure to add the icon first!).
```

"as we don't supply one" may come off as rude 😅

File: docs/tutorial/part-eight/index.md [hunk 2]
```diff
@@ -85,7 +85,7 @@ Quoting [Google](https://developers.google.com/web/fundamentals/web-app-manifest
 npm install --save gatsby-plugin-manifest
 ```
 
-2.  Add the plugin to the `plugins` array in your `gatsby-config.js` file.
+2.  Add the plugin to the `plugins` array in your `gatsby-config.js` file (don't forget to add the icon yourself as we don't supply one).
```
Comment (yeskunall) [inline docs/tutorial/part-eight/index.md]:
```suggestion
2.  Add the plugin to the `plugins` array in your `gatsby-config.js` file (make sure to add the icon first!).
```

Same here!

**Comment by nclandrei** [comment]
Thanks @yeskunall 🎉 Just applied the changes, sorry for the mistake!

**Comment by LekoArts** [comment]
Hello @nclandrei 👋 Do you have time to address Shannon's questions? Thanks!

**Comment by LekoArts** [comment]
Closing this out of inactivity. If anyone wants to continue, please have a look at https://github.com/gatsbyjs/gatsby/issues/8366 for details on what to do.

**Pull request closed without merging**